From daa58f5dd375e5df2b0245befc7825e0a1a7f6f9 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Mon, 25 May 2015 10:07:33 +0800
Subject: [PATCH] MLK-10963 ARM: dts: imx6ul-evk: change the dts file
 permission to 0x644

imx6ul-evk.dts file permission was changed by the commit: 2dcc7174e82fa.
Recover the file permission to 0x644.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 0 files changed

diff --git a/arch/arm/boot/dts/imx6ul-evk.dts b/arch/arm/boot/dts/imx6ul-evk.dts
old mode 100755
new mode 100644
-- 
1.8.0

